### DISCUSSION TITLE
New editor skin

### DISCUSSION THREAD

**Discussion by hydroper** [godotengine/godot-proposals#6282] [Editor]
The Godot Engine's code editor is pretty productive! I just wonder how it'd look if it adopted the Courier New font. Also you can't currently change the editor's font.

Look at this image at its resolution: [https://i.imgur.com/TEzkbbs.png](https://i.imgur.com/TEzkbbs.png)

**Comment by bossoverdose** [comment]
you can change the editor's font in editor settings

**Reply by hydroper** [reply]
I realized that now, but I'm having trouble with the font weight. It looks like it's set to "Light", why?

**Reply by Calinou** [reply]
The font weight depends on the font file you loaded. For system fonts, the default font weight should be picked, but it's possible that depending on your operating system, a light font weight is selected instead (I've had this issue on Linux).

As a workaround, copy the font file from the system fonts and specify its full path in the Editor Settings.

**Reply by hydroper** [reply]
Yep, I pointed exactly to the TTF.
